core/frontend/src/components/vehiclesetup/PwmSetup.vue

```diff
@@ -455,13 +482,25 @@ export default Vue.extend({
       this.selected_param = param
       this.edit_param_dialog = true
     },
+    parameterToUserFriendlyText(param: Parameter): string {
+      if (param.value === -1) { // GPIO
+        const servo_number = parseInt(/\d+/g.exec(param.name)?.[0] ?? '0', 10)
+        const gpio = this.servo_to_gpio[servo_number]
+        const param_using_this_gpio = this.gpio_to_parameter[gpio]
+        if (param_using_this_gpio) {
+          const pretty_name = param_using_this_gpio.name.split('_')[0].toLowerCase().toTitle()
+          return `GPIO [${pretty_name}]`
+        }
+        return `Unknown GPIO ${gpio}`
```

Comment:
```suggestion
        return `GPIO ${gpio} [Unknown]`
```
I think consistent formatting would be nicer. Starting with GPIO also makes it easier to find in the list.